python/ql/src/Security/CWE-770/MissingRateLimiting.qhelp

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
```

Comment:
I'm not sure what's caused this, but the qhelp preview has the title "Unknown query". I can see that the query has an `@name`, so I'm not sure why this hasn't been picked up.

Reply:
possibly it was due to the id being `python/...` instead of `py/...` which was incorrect? I've fixed this now so maybe it will work.